src/screens/FeedScreen/hooks/useCoreFeed.js

```diff
@@ -35,6 +29,12 @@ const useCoreFeed = () => {
   const {myProfile} = profileContext;
   const {bottom} = useSafeAreaInsets();
 
+  const {sendViewPostTimeWithFeeds, updateViewPostTime, isSamePostViewed} = useViewPostTimeHook(
+    dispatch,
+    timer,
+    viewPostTimeIndex
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useViewPostTimeHook` is being used here to get the functions `sendViewPostTimeWithFeeds`, `updateViewPostTime`, and `isSamePostViewed`. This is a good example of encapsulation and modularity where related functionalities are grouped together in a hook.
```diff
+ const {sendViewPostTimeWithFeeds, updateViewPostTime, isSamePostViewed} = useViewPostTimeHook(
+   dispatch,
+   timer,
+   viewPostTimeIndex
+ );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->